lib/commands/project.js

```diff
@@ -65,6 +65,76 @@ function sortImages(images) {
   return images;
 }
 
+project.resize = function (projectName, servoSize, cb) {
+  var projectId, projectName;
+
+  projectController.find({
+      userId : userConfig.data.userId
+    },
+    function (err, projects) {
+      if (err) {
+        err = error.handleApiError(err, 'GET_PROJECTS', cb);
+        if (err.length > 0) {
+          return cb(err);
+        }
+      }
+      if (projects.length === 0) {
+        modulus.io.error('You currently have no projects. One can be created using the create command.');
+        return cb();
+      }
+      project.cmd = "resize";
+
+      async.series([
+        function promptForProject(fn) {
+          project.chooseProjectPrompt(projects, projectName, function (err, result) {
+            if (err) {
+              return cb(err);
+            }
+
+            if (!result) {
+              return cb('You must select a project.');
+            }
+
+            projectId = result.id;
+            projectName = result.name;
+            projectDomain = result.domain;
+
+            fn();
+          });
+        },
+        function promptForServoSize(fn) {
+          if (servoSize && servoSize.length > 0) {
```

Comment:
I think this would allow the user to set whatever servoSize they want. Maybe see if their input matches an element in an array of possible sizes?

e.x (syntax may not be right)
```
if ([192, 512, 1024, 2048].indexOf(servoSize) !== -1) return fn()
```